bundles/org.openhab.binding.homeconnect/src/main/java/org/openhab/binding/homeconnect/internal/servlet/HomeConnectServlet.java

```diff
@@ -0,0 +1,628 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.homeconnect.internal.servlet;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.time.ZonedDateTime.now;
+import static org.apache.commons.lang.StringUtils.isEmpty;
```

Comment:
Please remove use of org.apache.commons.lang.StringUtils

Reply:
Yes, will do. One user reported it on the weekend. In some circumstances Apache Commons was missing in the OH3 environment.

Reply:
Now removed (included in my PR).